EasyPost.Tests/ErrorTest.cs

```diff
@@ -118,25 +121,42 @@ public void TestUnknownApiException5xxGeneration()
             HttpStatusCode statusCode = (HttpStatusCode)Enum.Parse(typeof(HttpStatusCode), unexpectedStatusCode.ToString());
             RestResponse response = new RestResponse { StatusCode = statusCode };
 
-            ApiError generatedError = ApiError.FromResponse(response);
+            ApiError generatedError = ApiError.FromErrorResponse(response);
 
-            // should be of base type ApiError
+            // the exception should be of base type ApiError
             Assert.Equal(typeof(ApiError), generatedError.GetType().BaseType);
-            // should be specifically of type UnexpectedHttpError
+            // specifically, the exception should be of type UnexpectedHttpError
             Assert.Equal(typeof(UnexpectedHttpError), generatedError.GetType());
         }
 
         [Fact]
         public void TestExceptionMessageFormatting()
         {
-            UseVCR("exception_message_formatting");
-
             Type type = typeof(Address);
             JsonError jsonError = new JsonDeserializationError(type);
 
             string expectedMessage = string.Format(Constants.ErrorMessages.JsonDeserializationError, type.FullName);
 
             Assert.Equal(expectedMessage, jsonError.Message);
         }
+
+        [Fact]
+        public void TestApiExceptionPrettyPrint()
+        {
+            const int statusCode = 401;
+
+            // Generate a dummy RestResponse with the given status code to parse
+            HttpStatusCode httpStatusCode = (HttpStatusCode)Enum.Parse(typeof(HttpStatusCode), statusCode.ToString());
+            RestResponse response = new RestResponse { StatusCode = httpStatusCode };
+
+            ApiError generatedError = ApiError.FromErrorResponse(response);
+
+            // unfortunately, we can't easily load error-related JSON data into the response for parsing, so the pretty print is going to fallback to default values.
```

Comment:
Can we not mock the return value of the called error? I don't love not being able to test something.

Reply:
In a way, the `TestBadParameters` test above this validates that error parsing works properly, since that makes a real, bad API call and evaluates the exception thrown by it.